src/webchat/store/connection/connection-handler.ts

```diff
@@ -3,9 +3,21 @@ import { setConnected, setReconnectionLimit } from "./connection-reducer";
 import { SocketClient } from "@cognigy/socket-client";
 
 export const registerConnectionHandler = (store: Store, client: SocketClient) => {
-    client.on('socket/connect', () => { store.dispatch(setConnected(true)) });
-    client.on('socket/reconnect', () => { store.dispatch(setConnected(true)) });
-    client.on('socket/disconnect', () => { store.dispatch(setConnected(false)) });
+    const handleConnected = () => {
+        store.dispatch(setConnected(true))
+    }
+
+    const handleDicsonnected = () => {
```

Comment:
"handleDicsonnected" unfortunate typo

Reply:
Please provide more details on how this is unfortunate

Reply:
https://en.wikipedia.org/wiki/Freudian_slip